contracts-periphery/Makefile

```diff
@@ -0,0 +1,39 @@
+# include .env file and export its env vars
+# (-include to ignore error if it does not exist)
+-include .env
+
+all: clean remove install update solc build 
+
+# Install proper solc version.
+solc:; nix-env -f https://github.com/dapphub/dapptools/archive/master.tar.gz -iA solc-static-versions.solc_0_8_10
+
+# Clean the repo
+clean  :; forge clean
+
+# Test your contract on a forked network
+test   :; forge test -f ${ETH_RPC_URL} --fork-block-number 10465092
+
+# Remove modules
+remove :; rm -rf ../.gitmodules && rm -rf ../.git/modules/* && rm -rf lib"
+
+# Install the Modules
+install :; 
+	forge install dapphub/ds-test 
+	forge install brockelmore/forge-std
+	forge install OpenZeppelin/openzeppelin-contracts
```

Comment:
These well delete all deps and update them to the latest main, which we don't want. We should delete `remove` and have `install` just run `forge install`

As a side note, did you install OZ by specifying a tag?